pages/community/styleguide/aboutguide.md

```diff
@@ -0,0 +1,35 @@
+# AsyncAPI Style Guide
+
+## Introduction
+Welcome to the AsyncAPI Style Guide. This page serves as a guideline to contributors on how to write, structure, and format content/documentation.
+
+These guidelines help us in maintaining language, tone, and voice throughout the website. It also makes it easier for multiple contributors to work together on a single doc while ensuring consistency, accuracy, and clarity. 
+
+## Writing Style
+Below are some recommendations to consider when writing your pages.
+
+### Have your target audience in mind
+Before you begin planning and constructing your content, it is important to know who you are writing for and how the readers will benefit from your content.
+This will help you in strategizing and simplifying your writing process.
+
+### Research
+Before creating your first draft, it is beneficial to research similar resources and use them as references. Avoid plagiarism or copying and pasting from other websites unless the content already exists on our site.
+
```

Comment:
Can we please add a paragraph/header for `Plagiarism` itself? I want us to actually define what plagiarism is, and provide a few examples of do/don't's. 

EX: actually compare two texts to explain what counts as plagiarism .. even add a scenario explaining how to cite  generated content from ChatGPT

Reply:
I have added the paragraph, how does it sound?

Reply:
Done